easybuild/easyconfigs/c/CMake/CMake-3.15.3-GCCcore-8.3.0.eb

```diff
@@ -0,0 +1,57 @@
+easyblock = 'ConfigureMake'
+
+name = 'CMake'
+version = '3.15.3'
+
+homepage = 'https://www.cmake.org'
+
+description = """
+ CMake, the cross-platform, open-source build system.  CMake is a family of
+ tools designed to build, test and package software.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
+
+source_urls = ['https://www.cmake.org/files/v%(version_major_minor)s']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['13958243a01365b05652fa01b21d40fa834f70a9e30efa69c02604e64f58b8f5']
+
+builddependencies = [
+    ('binutils', '2.32'),
+]
+
+dependencies = [
+    ('ncurses', '6.1'),
+    ('zlib', '1.2.11'),
+    ('bzip2', '1.0.8'),
+    ('cURL', '7.66.0'),
+    # OS dependency should be preferred if the os version is more recent then this version,
+    # it's nice to have an up to date openssl for security reasons
+    # ('OpenSSL', '1.1.1a'),
+]
+
+osdependencies = [
+    ('openssl-devel', 'libssl-dev', 'libopenssl-devel'),
+]
+
+configopts = '-- -DCMAKE_USE_OPENSSL=1 '
+configopts += '-DCURSES_INCLUDE_PATH=$EBROOTNCURSES/include '
+configopts += '-DCURSES_CURSES_LIBRARY=$EBROOTNCURSES/lib/libcurses.%s ' % SHLIB_EXT
+configopts += '-DCURSES_FORM_LIBRARY=$EBROOTNCURSES/lib/libform.%s ' % SHLIB_EXT
+configopts += '-DCURSES_NCURSES_LIBRARY=$EBROOTNCURSES/lib/libncurses.%s ' % SHLIB_EXT
+configopts += '-DCMAKE_USE_SYSTEM_BZIP2=1 '
+configopts += '-DBZIP2_INCLUDE_DIR=$EBROOTBZIP2/include '
+configopts += '-DBZIP2_LIBRARY_RELEASE=$EBROOTBZIP2/lib/libbz2.%s ' % SHLIB_EXT
+configopts += '-DCMAKE_USE_SYSTEM_ZLIB=1 '
+configopts += '-DZLIB_INCLUDE_DIR=$EBROOTZLIB/include '
+configopts += '-DZLIB_LIBRARY_RELEASE=$EBROOTZLIB/lib/libz.%s ' % SHLIB_EXT
+configopts += '-DCMAKE_USE_SYSTEM_CURL=1 '
+configopts += '-DCURL_INCLUDE_DIR=$EBROOTCURL/include '
+configopts += '-DCURL_LIBRARY=$EBROOTCURL/lib/libcurl.%s ' % SHLIB_EXT
+
+sanity_check_paths = {
+    'files': ["bin/%s" % x for x in ['ccmake', 'cmake', 'cpack', 'ctest']],
+    'dirs': [],
+}
+
```

Comment:
The PR by boegel has sanity_check_commands as well, which seems like a good idea. https://github.com/easybuilders/easybuild-easyconfigs/pull/8962/files